fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -597,11 +599,17 @@ private void computePartitionInfo() throws AnalysisException {
         }
         selectedPartitionNum = selectedPartitionIds.size();
 
+        String dbName = desc.getRef().getName().getDb();
+        long dbId = analyzer.getCatalog().getDb(dbName).map(Database::getId).orElse(0L);
+        long timestamp = System.currentTimeMillis();
+        VisitPartitionsInfo info = new VisitPartitionsInfo(dbId, olapTable.getId(), selectedPartitionIds, timestamp);
+        Catalog.getCurrentCatalog().getEditLog().logVisitPartition(info);
```

Comment:
bdbje cannot write log on non-master node. so if we wan't log this info, we either transfer all log request to master, or redirect all query to master.

Reply:
Since we can't redirect all queries to master, should I transfer all log request to master? Is there too many RPCs?